iFixit/react-commerce
# Add store home page hero section

closes #813 

## QA

1. Open Vercel preview
2. Visit [`/Store2`](https://react-commerce-git-add-store-home-page-hero-section-ifixit.vercel.app/Store2)
3. Verify that hero section is implemented as specified in #813 

## Post merge

This PR adds a new content type and a fix to gracefully handle missing content types on the remote server when seeding, so after having merged this we need to migrate the Strapi live server

cc @sterlinghirsh
